generated/python/grpc-google-cloud-logging-v2/setup.py

```diff
@@ -33,11 +33,11 @@
     'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: Implementation :: CPython',
   ],
-  description='GRPC library for the google-logging-v2 service',
+  description='GRPC library for the Google Logging service',
   long_description=open('README.rst').read(),
   install_requires=install_requires,
   license='Apache-2.0',
   packages=find_packages(),
-  namespace_packages=['google', 'google.logging', ],
+  namespace_packages=['google.cloud.grpc.logging', 'google.cloud.grpc', 'google.cloud', 'google'],
```

Comment:
Not sensitive. I've verified that the `grpc` and `gapic` packages load correctly together.